pkg/email/templates/mto_common_solution_poc_added_body.html

```diff
@@ -0,0 +1,31 @@
+{{template "shared_header.html"}}
+
+{{template "shared_mto_common_solution_header.html"}}
+
+<hr/>
+
+<p>A new point of contact has been added to {{.SolutionAcronym}}. You may double-check the details below.</p>
+
+<br/>
+
+<p><strong>Contact name:</strong> {{.ContactName}}</p>
+<p><strong>Contact email:</strong> {{.ContactEmail}}</p>
+<p><strong>Contact role:</strong> {{.ContactRole}}</p>
+<p><strong>Primary contact:</strong> {{.IsPrimary}}</p>
+<p><strong>Will receive notifications from MINT:</strong> {{.WillReceiveNotifications}}</p>
+
+<br/>
+
+<h2>
+  <a href="{{.ClientAddress}}/help-and-knowledge/operational-solutions?solution-key={{.Key}}" target="_blank">View operational solution in MINT</a>
+</h2>
+
+<br/>
+
+<p>If you believe this to be an error, please check with the operational solution team. You may edit point of contact information in MINT if it is incorrect.</p>
+
+<hr/>
+
+{{template "shared_mto_common_solution_header.html"}}
```

Comment:
I see most of these emails use the shared header again at the end of the email, I don't see that in FIGMA. Did you mean to reference another shared template? Footer etc?